themes/scss/widgets/canvas.scss

```diff
@@ -71,12 +67,13 @@
 // Widget media query
 @include media-query() {
   .CDB-Widget-canvas {
-    width: calc(100% - $baseSize * 11);
-    min-width: calc(100% - $baseSize * 11);
-    max-width: calc(100% - $baseSize * 11);
+    $w: calc(100% - #{$baseSize} * 9);
```

Comment:
Name of variable `baseSize` should be written in all lowercase letters with hyphens instead of underscores